java/java-v754/src/main/java/org/cloudburstmc/protocol/java/v754/serializer/play/clientbound/SetPassengersSerializer_v754.java

```diff
@@ -0,0 +1,33 @@
+package org.cloudburstmc.protocol.java.v754.serializer.play.clientbound;
+
+import com.nukkitx.protocol.exception.PacketSerializeException;
+import io.netty.buffer.ByteBuf;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import org.cloudburstmc.protocol.java.JavaPacketHelper;
+import org.cloudburstmc.protocol.java.JavaPacketSerializer;
+import org.cloudburstmc.protocol.java.packet.play.clientbound.SetPassengersPacket;
+
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class SetPassengersSerializer_v754 implements JavaPacketSerializer<SetPassengersPacket> {
+    public static final SetPassengersSerializer_v754 INSTANCE = new SetPassengersSerializer_v754();
+
+    @Override
+    public void serialize(ByteBuf buffer, JavaPacketHelper helper, SetPassengersPacket packet) throws PacketSerializeException {
+        helper.writeVarInt(buffer, packet.getEntityId());
+        helper.writeVarInt(buffer, packet.getPassengerIds().length);
```

Comment:
There is a `writeArray` method in JavaPacketHelper which allows you to write this much easier.